control-plane/rest/service/src/v0/nodes.rs

```diff
@@ -56,6 +58,41 @@ impl apis::actix_server::Nodes for RestApi {
         let node = client().drain(id.into(), label).await?;
         Ok(node.into())
     }
+
+    async fn put_node_label(
+        Path((id, label)): Path<(String, String)>,
+    ) -> Result<models::Node, RestError<RestJsonError>> {
+        let mut label_map = HashMap::new();
+        let mut iter = label.split('=');
+        if let Some(key) = iter.next() {
+            if let Some(value) = iter.next() {
+                label_map.insert(key.trim().to_string(), value.trim().to_string());
```

Comment:
Seems like this add a single label only?
Elsewhere in the pr it seemed put would add a list of labels hmm